flags/change-flag.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/sh
```

Comment:
This script just swaps the value of the flag in `changing-flag.json` once a second.